cms/models.py

```diff
@@ -26,6 +26,12 @@ def programs_for_sign_up(programs):
     return [ProgramSerializer().to_representation(p) for p in programs]
 
 
+def faculty_for_carousel(faculty):
+    """formats faculty info for the carousel"""
+    from cms.serializers import FacultySerializer
```

Comment:
Unfortunately, no. That would cause a circular import problem. (`serializers.py` importing `models.py` importing `serializers.py`...)